x-pack/heartbeat/monitors/browser/synthexec/enrich_test.go

```diff
@@ -111,6 +126,82 @@ func TestJourneyEnricher(t *testing.T) {
 				})
 			}
 		})
+
+	}
+
+	for idx, se := range synthEvents {
+		e := &beat.Event{}
+		stdFields.IsInline = true
+		t.Run(fmt.Sprintf("inline monitor event %d", idx), func(t *testing.T) {
+			je.enrich(e, se, stdFields)
+			testslike.Test(t, lookslike.MustCompile(common.MapStr{
+				"monitor.id":   stdFields.Id,
+				"monitor.name": stdFields.Name,
+			}), e.Fields)
+
+			sv, _ := e.Fields.GetValue("suite")
+			require.Nil(t, sv)
+		})
+	}
+}
+
+func TestEnrichConsoleSynthEvents(t *testing.T) {
+	tests := []struct {
+		name  string
+		je    *journeyEnricher
+		se    *SynthEvent
+		check func(t *testing.T, e *beat.Event, je *journeyEnricher)
+	}{
+		{
+			"stderr",
+			&journeyEnricher{},
+			&SynthEvent{
+				Type: "stderr",
+				Payload: common.MapStr{
+					"message": "Error from synthetics",
+				},
+			},
+			func(t *testing.T, e *beat.Event, je *journeyEnricher) {
+				v := lookslike.MustCompile(common.MapStr{
```

Comment:
Would there be any value in also testing the `monitor.*` values that are set here? Also, as a note, you can use `lookslike.Strict` to ensure that there are no extra fields if that's appropriate. It just wraps the outer `lookslike.MustCompile`, but you do have to define every field.

Reply:
Not sure if there would be any value, I added it to denote the case when there are no journeys ran and this was the only message from console.